lib/ocpp/v201/charge_point.cpp

```diff
@@ -4041,17 +4042,15 @@ void ChargePoint::execute_change_availability_request(ChangeAvailabilityRequest
 }
 
 // K01.27 - load profiles from database
```

Comment:
Comment is outdated

Reply:
Removed this and the other comment you mentioned